### ISSUE TITLE
careful to pick up defaults when we bypass a function

### ISSUE THREAD

**Issue by jordansread** [USGS-R/gsplot#236]
Skipping defaults w/ values (e.g., for plot.xy lty = par("lty")) doesn't matter because `graphics::function` is called ultimately, and those defaults get in there. 

but...bypassing functions like `curve` to create a line call doesn't actually grab the defaults of `curve` (e.g., `n=101`).

We should work those in whenever we bypass.

**Comment by lindsayplatt** [comment]
@jread-usgs can you give an example? Not sure how to test if this is fixed already or not.

**Comment by jordansread** [comment]
yes, our implementation of `curve` was an example of this. But you fixed it: https://github.com/USGS-R/gsplot/commit/7069295685c2825b45af9baeb53731cdc7e7c540

**Comment by lindsayplatt** [comment]
Ah I see, this is case by case. So something we need to keep in mind if we add something that bypasses. Going to leave this open so that we can verify that we don't have anymore occurrences, but not going to add to milestone 0.8.0
